lib/kb_ObjectInfo/CreateFasta_Report.py

```diff
@@ -1,8 +1,9 @@
 import time
+import logging
 
 def log(message, prefix_newline=False):
```

Comment:
As noted in several other places, this `log` function should be defined once in the codebase, in a separate file, and imported into each file which needs it.

Also, I'm wondering about the logging at "info" level. I don't know how app logging levels are controlled (as far as I know a user cannot select which logging level to view in the Narrative). Default for Python is WARNING level, so logging all of this at info is probably quite safe.

Some of the logging seems quite detailed and should perhaps be logged at DEBUG level.

Some of the logging calls seem to be actual warnings about problems with data, and should be logged at WARNING level.

There are comments associated with some of the logging lines which indicate that someone thought they were excessive and should only be displayed while debugging, which I would interpret as meaning they should be logged at DEBUG level.

To support the above, the `log()` function should have a "level" parameter which would default to the most commonly or safest debug level, probably "info".

Reply:
This example was taken from other modules and isn't causing any error. It falls more in the category of improvement and can be postponed. All comments will be noted in the improvement ticket https://kbase-jira.atlassian.net/browse/PTV-1860.